packages/node_modules/@ciscospark/test-helper-test-users/src/index.js

```diff
@@ -156,5 +156,9 @@ module.exports = {
     }
 
     return _remove(users);
-  }
+  },
+
+  // Org for Federation test users
+  EU_PRIMARY_ORG_ID: 'c1b781f7-b70b-43f7-bac6-6590d98d50fa'
```

Comment:
Is this okay to put here, or should this be in `.env`?

Reply:
Moved to env file

Reply:
@glenatwork is there a reason not to put it in `.env.default`?